### PR TITLE
Language dropdown

### PR DESCRIPTION
**Pull request by TheoChevalier**
Slightly underestimated the amount of code needed… Most of it comes from an example I’ve found online, but it looked solid.

The last missing part is displaying it only `if (location.pathname == "/")` but I’m missing react knowledge on why this condition can’t be added in the render function.

Also, don’t look at the individual commits, they are meaningless, I’ll squash them all when merging.

In dropDownOnChange, we can probably use a React Router redirect, but I just don’t know how this works…

### CODE REVIEW

**Comment by ScottDowne** [comment]
I'm not super concerned with using react router for redirects. I mostly just like it as an easy way to store the page map as html.

File: less/components/nav.less [hunk 1]
```diff
@@ -39,6 +53,8 @@
       height: 65px;
       display: flex;
       align-items: center;
+      justify-content: flex-start;
+      flex-direction: row;
```
Comment (ScottDowne) [inline less/components/nav.less]: Is this still needed?

Comment (ScottDowne) [inline less/components/nav.less]: Both of these ^^

If I remove them locally, everything seems to look the same.

File: less/components/nav.less [hunk 2]
```diff
@@ -23,6 +23,20 @@
         opacity: 1;
       }
     }
+    .nav-lang-selector {
+      position: absolute;
```
Comment (ScottDowne) [inline less/components/nav.less]: You can accomplish a right align inside a flex box using this `margin-left: auto;` It's looks pretty arbitrary.

Comment (ScottDowne) [inline less/components/nav.less]: So I think you can remove the position, right, top, and static height 65 on select.

File: less/components/nav.less [hunk 3]
```diff
@@ -23,6 +23,20 @@
...
+      right: 0;
+      top: 0;
+      select {
+        border: none;
+        box-shadow: none;
+        background-color: transparent;
+        height: 65px;
+        font-size: 14px;
+        font-family: Arvo;
+        font-weight: 600;
```
Comment (ScottDowne) [inline less/components/nav.less]: Should probably put a `cursor: pointer;` on here.

**Comment by ScottDowne** [comment]
@TheoChevalier I added some comments for how would could simplify some of the css/react, and some small stuff.

I also think you could reduce the complexity of the component a bit, but not sure if it's necessary, as it works quite well. I think mostly with some work we could kill the need for `componentWillReceiveProps` but I'm really thinking it's not worth it at this point, so it would mostly be for your own benefit.

File: less/components/nav.less (hunk 2 above)
Comment (TheoChevalier) [inline less/components/nav.less]: I initially used `margin-left: auto;`, but it doesn’t work in the mobile view. Do you prefer still using flexbox for desktop and position absolute for mobile?

**Comment by ScottDowne** [comment]
Looks like you got them all. Thanks! Good work.

**Pull request merged**